Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: bump itertools dependency from 0.9 to 0.10

itertools upstream's CHANGELOG.md says that no significant API changes
happened between 0.9 and 0.10 (just a shift in the MSRV, which is fine
for debian).

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -44,7 +44,7 @@
 features = ["wrap_help"]
 
 [dependencies.itertools]
-version = "0.9"
+version = "0.10"
 
 [dependencies.rpassword]
 version = "5.0"
Bump-itertools.patch
